optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/constraintweight/descriptor/ConstraintConfigurationDescriptor.java

```diff
@@ -78,7 +79,8 @@ public String getConstraintPackage() {
     // Lifecycle methods
     // ************************************************************************
 
-    public void processAnnotations(DescriptorPolicy descriptorPolicy, ScoreDefinition scoreDefinition) {
+    public void processAnnotations(DomainAccessType domainAccessType, DescriptorPolicy descriptorPolicy,
+            ScoreDefinition scoreDefinition) {
```

Comment:
the acess type last

Reply:
DomainAccessType is merged into DescriptorPolicy per discussion